tests/common/tensor_test.py

```diff
@@ -94,3 +97,110 @@ def test_sort_tensor_by_length(self):
 
         # Test restoration indices correctly recover the original tensor.
         assert sorted_tensor[reverse_indices].equal(tensor)
+
+    def test_masked_softmax_no_mask(self):
```

Comment:
It's actually probably a bit excessive...  =).  I just copied it from DeepQA - Nelson wrote them original, and I just translated it to pytorch.